react/components/SideBarItem.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component, Fragment } from 'react'
+import PropTypes from 'prop-types'
+import { IconCaretDown, IconCaretUp } from 'vtex.styleguide'
+
+export default class SideBarItem extends Component {
+  static propTypes = {
+    /** Sidebar's item. */
+    item: PropTypes.object.isRequired,
+    /** Link values to create the redirect. */
+    linkValues: PropTypes.arrayOf(PropTypes.string).isRequired,
+    /** Closes sidebar. */
+    onClose: PropTypes.func.isRequired,
+  }
+
+  static contextTypes = {
```

Comment:
Use the `withRuntimeContext` hoc, and get the `navigate` function from the prop `runtime`, instead of using the old react context API